stdlib/urllib/request.pyi

```diff
@@ -155,6 +155,8 @@ class HTTPPasswordMgrWithPriorAuth(HTTPPasswordMgrWithDefaultRealm):
 
 class AbstractBasicAuthHandler:
     rx: ClassVar[Pattern[str]]  # undocumented
+    passwd: Union[HTTPPasswordMgr, None]
```

Comment:
This can't be `None`. If `None` is passed to `__init__`, then it makes a new `HTTPPasswordMgr` instance and uses it instead.

Reply:
Agree, so it is only of type `HTTPPasswordMgr`, right?